hydra-core/lib/generators/hydra/head_generator.rb

```diff
@@ -111,5 +111,13 @@ def inject_hydra_user_behavior
         puts "     \e[31mFailure\e[0m  Hydra requires a user object in order to apply access controls. This generators assumes that the model is defined in the file #{file_path}, which does not exist.  If you used a different name, please re-run the generator and provide that name as an argument. Such as \b  rails -g hydra:head client"
       end
     end
+    
+    def create_wrapper_configuration
```

Comment:
Should we move this to a new generator (and still call it here), such that people who've already generated hydra can easily run this without other side-effects?